pkg/config/v1alpha1/types.go

```diff
@@ -57,6 +57,10 @@ type ControllerManagerConfigurationSpec struct {
 	// The graceful shutdown is skipped for safety reasons in case the leader election lease is lost.
 	GracefulShutdownTimeout *metav1.Duration `json:"gracefulShutDown,omitempty"`
 
+	// PrestartTimeout is the duration given to each prestart hook before they return successfully.
+	// To use prestart hooks without a timeout, set to a negative duration, e.g. time.Duration(-1)
+	PrestartTimeout *metav1.Duration `json:"prestartTimeout,omitempty"`
```

Comment:
Remove this from these types, they've been deprecated and will be removed

Reply:
That's really annoying, since we had just finishing migrating _to_ them, as that's what all the examples showed.